routes/api/v1/favorites.js

```diff
@@ -0,0 +1,85 @@
+var express = require('express');
+var router = express.Router();
+
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../../../knexfile')[environment];
+const database = require('knex')(configuration);
+const fetch = require('node-fetch');
+
+
+router.post('/', (request, response) => {
+  // 1. check request body comes with location and api_key
+  const info = request.body;
+
+  for (let requiredParameter of ['location', 'api_key']) {
+    if (!info[requiredParameter]) {
+      return response
+        .status(422)
+        .send({ error: `Expected format: { location: <String>, api_key: <String> }. You're missing a "${requiredParameter}" property.` });
+    }
+  }
+  // 2. check api_key in users and return user
+  database('users')
+    .where('api_key', info['api_key'])
+    .select()
```

Comment:
I believe you can get rid of this select portion

Reply:
Correct!